### ISSUE TITLE
In 8.11.0 DataTable can't be instantiated if it is not selectable and doesn't contain CheckBox column

### ISSUE THREAD

**Issue by pekour** [sebfz1/wicket-jquery-ui#329]
Hi. It looks like `com.googlecode.wicket.kendo.ui.datatable.DataTableBehavior#newOnChangeAjaxBehavior` makes `DataTable` constantly failing to initialize when it is not selectable and has no `CheckboxColumn`:

```java.lang.IllegalArgumentException: Argument 'behavior' may not be null.
     at org.apache.wicket.util.lang.Args.notNull(Args.java:41)
     at org.apache.wicket.Behaviors.add(Behaviors.java:46)
     at org.apache.wicket.Component.add(Component.java:4481)
     at com.googlecode.wicket.kendo.ui.datatable.DataTableBehavior.bind(DataTableBehavior.java:122)
     at org.apache.wicket.Behaviors.add(Behaviors.java:56)
     at org.apache.wicket.Component.add(Component.java:4481)
     at com.googlecode.wicket.kendo.ui.datatable.DataTable.onInitialize(DataTable.java:341)
```

**Comment by sebfz1** [comment]
Hi Alexander, thanks for reporting!

CheckboxColumn is indeed a new feature. Maybe I missed something
somewhere... I'll look at it asap and publish a patch release.

**Comment by pekour** [comment]
Thank you!

**Comment by sebfz1** [comment]
Released `8.11.1`

**Comment by mdbergmann** [comment]
Hi @sebfz1

Is this fix also in Wicket 9 branch?
I'm seeing similar issues in tests with wicket-kendo-ui:9.2.1

```
  java.lang.IllegalArgumentException: Argument 'behavior' may not be null.
  at org.apache.wicket.util.lang.Args.notNull(Args.java:41)
  at org.apache.wicket.Behaviors.add(Behaviors.java:46)
  at org.apache.wicket.Component.add(Component.java:4496)
  at com.googlecode.wicket.kendo.ui.datatable.DataTableBehavior.bind(DataTableBehavior.java:122)
  at org.apache.wicket.Behaviors.add(Behaviors.java:56)
  at org.apache.wicket.Component.add(Component.java:4496)
  at com.googlecode.wicket.kendo.ui.datatable.DataTable.onInitialize(DataTable.java:341)
```

**Comment by sebfz1** [comment]
Yes the fix is in the wicket 9 branch.
I will release 9.3.0 asap

**Comment by sebfz1** [comment]
Released `9.3.0`

**Comment by EveryonePrint-dev** [comment]
Cool, it works.
Thanks
